spec/requests/api/v1/posts/index_spec.rb

```diff
@@ -30,4 +30,37 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe "POST /api/v1/posts" do
+
+    describe "successfully" do
+      before do
+        post "/api/v1/posts", params: {
+          post: {
+            image: ('spec/fixtures/dummy_image.jpg'),
+            caption: 'Lorem ipsum dolor',
+            category: 'work',
+            longitude: 53.06,
+            latitude: 18.03,
+            }
+        }, headers: headers
+      end
+
+      it "creates a new post" do
+        expect(json_response["message"]).to eq "Successfully created"
+        expect(response.status).to eq 200
+        
+      end
+
+      it "sends back into the response the newly created post information" do
+        post = Post.last
+        expect(json_response["id"]).to eq post.id
```

Comment:
you need to test that the image you send in is being attached to the post

Reply:
done